Need to define GOPATH to enable Terraform / Ansible to build Cloud API Adaptor

This follows on from [kata-agent build failure due to libseccomp #206](https://github.com/confidential-containers/cloud-api-adaptor/issues/206) which was fixed by PR [ibmcloud: Build static libseccomp for kata-agent #207](https://github.com/confidential-containers/cloud-api-adaptor/pull/207)

The Terraform / Ansible build of CAA now fails with: -

```text
�[0m�[1mmodule.podvm_build.null_resource.ansible (local-exec):�[0m �[0mfatal: [10.244.64.4]: FAILED! => {"changed": true, "cmd": ". ~/.bashrc\nmake build\n", "delta": "0:00:43.314509", "end": "2022-09-10 08:24:00.621071", "msg": "non-zero return code", "rc": 2, "start": "2022-09-10 08:23:17.306562", "stderr": "/bin/sh: 13: /root/.bashrc: shopt: not found\n/bin/sh: 21: /root/.bashrc: shopt: not found\n/bin/sh: 102: /root/.bashrc: source: not found\n../../../kata-containers/ci/lib.sh: line 9: GOPATH: unbound variable\nmake: *** [/root/cloud-api-adaptor/ibmcloud/image/staticlib/kata-libseccomp/lib/libseccomp.a] Error 1", "stderr_lines": ["/bin/sh: 13: /root/.bashrc: shopt: not found", "/bin/sh: 21: /root/.bashrc: shopt: not found", "/bin/sh: 102: /root/.bashrc: source: not found", "../../../kata-containers/ci/lib.sh: line 9: GOPATH: unbound variable", "make: *** [/root/cloud-api-adaptor/ibmcloud/image/staticlib/kata-libseccomp/lib/libseccomp.a] Error 1"], "stdout": "curl -s -L -o \"focal-server-cloudimg-s390x.img\" \"https://cloud-images.ubuntu.com/focal/current/focal-server-cloudimg-s390x.img\"\ncd \"../..\" && make agent-protocol-forwarder\nmake[1]: Entering directory '/root/cloud-api-adaptor'\nCGO_ENABLED=0 go build -tags=ibmcloud -o \"agent-protocol-forwarder\" \"cmd/agent-protocol-forwarder/main.go\"\nmake[1]: Leaving directory '/root/cloud-api-adaptor'\ninstall -D --compare \"../../agent-protocol-forwarder\" \"files/usr/local/bin/agent-protocol-forwarder\"\n../../../kata-containers/ci/install_libseccomp.sh /root/cloud-api-adaptor/ibmcloud/image/staticlib/kata-libseccomp /root/cloud-api-adaptor/ibmcloud/image/staticlib/kata-gperf\nMakefile:92: recipe for target '/root/cloud-api-adaptor/ibmcloud/image/staticlib/kata-libseccomp/lib/libseccomp.a' failed", "stdout_lines": ["curl -s -L -o \"focal-server-cloudimg-s390x.img\" \"https://cloud-images.ubuntu.com/focal/current/focal-server-cloudimg-s390x.img\"", "cd \"../..\" && make agent-protocol-forwarder", "make[1]: Entering directory '/root/cloud-api-adaptor'", "CGO_ENABLED=0 go build -tags=ibmcloud -o \"agent-protocol-forwarder\" \"cmd/agent-protocol-forwarder/main.go\"", "make[1]: Leaving directory '/root/cloud-api-adaptor'", "install -D --compare \"../../agent-protocol-forwarder\" \"files/usr/local/bin/agent-protocol-forwarder\"", "../../../kata-containers/ci/install_libseccomp.sh /root/cloud-api-adaptor/ibmcloud/image/staticlib/kata-libseccomp /root/cloud-api-adaptor/ibmcloud/image/staticlib/kata-gperf", "Makefile:92: recipe for target '/root/cloud-api-adaptor/ibmcloud/image/staticlib/kata-libseccomp/lib/libseccomp.a' failed"]}
```

because `GOPATH` hasn't been set.

The issue has been raised to address that.

--- a/ibmcloud/terraform/cluster/ansible/kube-playbook.yml
+++ b/ibmcloud/terraform/cluster/ansible/kube-playbook.yml
@@ -136,6 +136,17 @@
         executable: /bin/bash
         creates: /usr/local/go/bin/go
 
+    - name: Create GOPATH
+      shell: |
+        mkdir -p /root/go
+
+        if ! grep -q '^GOPATH=/root/go' /root/.bashrc; then
+          echo 'export GOPATH=/root/go' >> /root/.bashrc
+        fi
+      args:
+        executable: /bin/bash
+        creates: /root/go
+        
     - name: Install containerd
       shell: |
         set -o errexit -o pipefail

